dali/test/python/operator_2/test_resize.py

```diff
@@ -803,6 +804,45 @@ def test_very_small_output():
             yield _test_very_small_output, dim, device
 
 
+large_data = None
+large_data_resized = None
+
+
+@params((types.INTERP_NN, False), (types.INTERP_LINEAR, False), (types.INTERP_LINEAR, True))
+def test_large_gpu(interp, antialias):
+    def make_cube(d, h, w):
+        z = np.arange(d)[:, np.newaxis, np.newaxis]
+        z = (z * 256 / z.size).astype(np.uint8)
+        z = np.stack([z, np.zeros_like(z), np.zeros_like(z)], axis=3)
+        y = np.arange(h)[np.newaxis, :, np.newaxis]
+        y = (y * 256 / y.size).astype(np.uint8)
+        y = np.stack([np.zeros_like(y), y, np.zeros_like(y)], axis=3)
+        x = np.arange(w)[np.newaxis, np.newaxis, :]
+        x = (x * 256 / x.size).astype(np.uint8)
+        x = np.stack([np.zeros_like(x), np.zeros_like(x), x], axis=3)
+        return x + y + z
+
+    global large_data
```

Comment:
can't nose2 run tests in parallel?

Reply:
@mzient , you could go with class-based suite and "static" setUpClass method to prepare data once in more nose2 compliant way. I think I've done that in debayer tests if you're looking for convienient copy-paste. :)

Reply:
@jantonguirao The only danger would be that the caching wouldn't work
@stiepan Perhaps, I've never done that.

Reply:
I'd go with the idea that @stiepan suggests

Reply:
I'll do it in a follow-up. This is a fix for a hard fault and merging it is more important than code style issues _in tests_.